PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawGraph.java

```diff
@@ -87,36 +86,35 @@ private void drawEdges(Pane pane, HashMap<Integer, Node> nodes,
         pane.getChildren().add(edge);
         edge.toBack();
         edge.setSmooth(true);
-
-        //        QuadCurve curve = new QuadCurve();
-        //
-        //        curve.setFill(Color.TRANSPARENT);
-        //        curve.setStroke(Color.BLACK);
-        //        curve.setStrokeWidth(2.0d);
-        //
-        //        fromCircle.centerXProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.startXProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //        fromCircle.centerYProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.startYProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //        toCircle.centerXProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.endXProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //        toCircle.centerYProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.endYProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //
-        //        pane.getChildren().add(curve);
-        //        curve.toBack();
       }
     });
   }
 
+  //        QuadCurve curve = new QuadCurve();
```

Comment:
Thomas quite specifically asked for curved lines, so any curvature is
better than none I would say

On Tue, 10 May 2016 at 23:38 Faris Elghlan <notifications@github.com> wrote:

> In
> PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawGraph.java
> <https://github.com/ProgrammingLife2016/PL2-2016/pull/36#discussion_r62757374>
> :
>
> >        }
> >      });
> >    }
> >
> > +  //        QuadCurve curve = new QuadCurve();
>
> This was added during pull request #27
> <https://github.com/ProgrammingLife2016/PL2-2016/pull/27>. I didn't
> really agree with merging this code, since it is not usefull yet (as I had
> also stated in my last comment on that pull request), since the lines were
> just bend arbitrarily and didn't avoid collision, so I just commented the
> code (and moved it out of the method to make the method smaller, which
> maybe caused the git blame to be incorrect?).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ProgrammingLife2016/PL2-2016/pull/36/files/591dc29426a7a5836160d3f1d94aa25480453a46..c33ea115450bfcb1ff1b9795fdd1435a1ac617b0#r62757374>
>